stable/README.md

```diff
@@ -1,167 +1,167 @@
+Assign the correct SecurityContextConstraints to the _nuodb_ and _default_ service accounts
 
 ```bash
 oc apply -f deploy/nuodb-scc.yaml -n nuodb
 oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:nuodb -n nuodb
 oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:default -n nuodb
 ```
 
-Here is a short list of NuoDB charts and their privilege requirements.
+## Confirm that the Helm client and Tiller server are able to communicate correctly. 
 
-| Charts | Privilege | Short Explanation |
-| ----- | ----------- | ------ |
-| transparent-hugepage| allowHostDirVolumePlugin: true | To mount hostPath and disable THP on host|
-| transparent-hugepage| volumes.hostPath | To mount hostPath and disable THP on host|
-| transparent-hugepage| seLinuxContext.* | To mount hostPath and disable THP on host|
-| admin, database| allowedCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
-| admin, database| defaultAddCapabilities.FOWNER | To change directory ownership in PV to the nuodb process|
+The results should be as follows:
 
+```bash
+$ helm version
+Client: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+Server: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
+```
+
+
+# Deploying NuoDB using Helm Charts
+
+The following section outlines the steps in order to deploy a NuoDB database using this Helm Chart repository.
+
+## Configuration Parameters
 
-### Deploy NuoDB (et al) via Helm Charts
+Each Helm Chart has a default values.yaml parameter file that contains configuration parameters specific to that chart. The configuration is structured where configuration values are implemented following a single-definition rule, that is, values are structured and scoped, and shared across charts; e.g. for admin, its parameters are specified once in a single values file which is used for all the charts, and the database chart can use admin values for configuring connectivity of engines to a specific admin process. The same goes for other values **shared** amongst Helm charts. A few key points here:
 
-    IMPORTANT:
-    
-    You MUST first disable THP on nodes where NuoDB will run. Run the `transparent-hugepage` chart first.
+- values files have structure, values are scoped
+- different values files for different deployments
+- values files follow the single definition rule (no repeats)
+- global configuration exists under its own scoped section
+- each chart has its own scoped section named after it
+- cloud information is used to drive availability zones (particularly)
 
-In a nutshell the order of installation is:
+## Deployment Steps
+
+**Note:** You MUST first disable Linux Transparent Huge Pages(THP) on all cluster nodes that will host NuoDB pods. Run the `transparent-hugepage` chart first.
```

Comment:
this wording is overly aggressive. THP must be disabled, but it MUST not be done via this chart. It can be done by any other means such as `tuned profiles`. If THP is disabled in a different way, this chart is not required.

Reply:
Do we offer tuned recommendation and how to elsewhere? This is what we currently have documented. If needed we can enhance this section in a future PR if really needed.

Reply:
it was the same wording as before. Ok. Resolving this comment